### PR TITLE
lint, fix tests, make compatible with current Django, use variable to hold posting url

### PR DESCRIPTION
**Pull request by johanneswilm**

### CODE REVIEW

**Comment by johanneswilm** [comment]
@Natim This is my work so far and it's all working.

What I was working on in a different branch is adding frontend testing and then after that I thought I'd try to update to using the JavaScript `fetch` API. But that can be another day/another PR.

File: demo/demoproject/manage.py [hunk 1]
```diff
@@ -11,7 +11,7 @@ def main():
         raise ImportError(
             "Couldn't import Django. Are you sure it's installed and "
             "available on your PYTHONPATH environment variable? Did you "
-            "forget to activate a virtual environment?"
+            "forget to activate a virtual environment?",
```
Comment (Natim) [inline demo/demoproject/manage.py:14]:
```suggestion
            "forget to activate a virtual environment?"
```

File: manage.py [hunk 2]
```diff
@@ -10,6 +10,6 @@
         raise ImportError(
             "Couldn't import Django. Are you sure it's installed and "
             "available on your PYTHONPATH environment variable? Did you "
-            "forget to activate a virtual environment?"
+            "forget to activate a virtual environment?",
```
Comment (Natim) [inline manage.py:13]:
```suggestion
            "forget to activate a virtual environment?"
```

File: demo/demoproject/manage.py (hunk 1 above)
Comment (Natim) [reply demo/demoproject/manage.py:14]: If you can that would be great because the multiline is for the string rather than multiline params in that case.

Comment (johanneswilm) [reply demo/demoproject/manage.py:14]: I know, but it's still a single argument in a list of arguments. And that's when a lot of Python-people prefer having a comma at the end. Let me dig into whether it is possible to remove it.

Comment (johanneswilm) [reply demo/demoproject/manage.py:14]: Yeah, so it is this rule that is the cause: https://github.com/asottile/add-trailing-comma

And it did correctly understand that it is a multiline string. It's just that it always sets a comma after the last function argument, even if there is just one. You can just disable that one if you don't want those commas, but it's quite correct to have them.

Comment (johanneswilm) [reply demo/demoproject/manage.py]: @Natim look at the last commit of this PR. It removes it.

Comment (Natim) [reply demo/demoproject/manage.py]: Thank you!

**Pull request merged**